history/provinces/00_k_lordaeron.txt

```diff
@@ -372,9 +384,16 @@
 	culture = lordaeronian
 	religion = holy_light
 	holding = castle_holding
+	100.1.1 = {
+		special_building_slot = scarlet_monastery_01
+	}
+	220.1.1 = {
+		special_building = scarlet_monastery_01
+	}
 
 	605.9.9={
 		religion = scarlet_light
+		special_building = scarlet_monastery_02
```

Comment:
I checked the map, I think the best for that is c_maplechill...

Reply:
But the wiki says it's located in the [Whispering Gardens](https://warcraft.wiki.gg/wiki/Whispering_Gardens) https://warcraft.wiki.gg/wiki/Scarlet_Monastery

Reply:
Badly rendered on the map sadly.
Whisperings gardens are like... Under the Monastery.